rofs/cache.go

```diff
@@ -8,10 +8,16 @@ import (
 	"syscall"
 
 	"github.com/threefoldtech/0-fs/meta"
+	"github.com/threefoldtech/0-fs/storage"
 )
 
-func (fs *filesystem) path(hash string) string {
-	base := fs.cache
+type Cache struct {
+	Cache   string
+	Storage storage.Storage
```

Comment:
Those fields don't need to be public. On the contrary they are better be private